engine/source/runtime/function/render/render_type.h

```diff
@@ -120,14 +121,13 @@ namespace Pilot
                    m_metallic_roughness_file == rhs.m_metallic_roughness_file && m_normal_file == rhs.m_normal_file &&
                    m_occlusion_file == rhs.m_occlusion_file && m_emissive_file == rhs.m_emissive_file;
         }
+
         size_t getHashValue() const
         {
-            size_t h0 = std::hash<std::string> {}(m_base_color_file);
-            size_t h1 = std::hash<std::string> {}(m_metallic_roughness_file);
-            size_t h2 = std::hash<std::string> {}(m_normal_file);
-            size_t h3 = std::hash<std::string> {}(m_occlusion_file);
-            size_t h4 = std::hash<std::string> {}(m_emissive_file);
-            return (((h0 ^ (h1 << 1)) ^ (h2 << 1)) ^ (h3 << 1)) ^ (h4 << 1);
+            size_t hash = 0;
+            hash_combine(
+                hash, m_base_color_file, m_metallic_roughness_file, m_normal_file, m_occlusion_file, m_emissive_file);
```

Comment:
It seems that the formatting configure is wrong here, I'll submit a manual formatted version.